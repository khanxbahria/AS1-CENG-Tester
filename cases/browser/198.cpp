#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("ll", "cc","dd");
Tab B = Tab("xxx", "zz","p");
Tab C = Tab("iii", "jj","tt");
Tab D = Tab("sss", "j","r");
Tab E = Tab("sss", "uu","jj");
Tab F = Tab("jj", "ss","iii");
Tab G = Tab("ll", "g","xxx");
Tab H = Tab("aaa", "d","h");
Tab I = Tab("ll", "uu","eee");
Tab J = Tab("uu", "ooo","ee");
Tab K = Tab("ff", "nn","d");
Tab L = Tab("yy", "ddd","fff");
Tab M = Tab("uu", "sss","q");
Tab N = Tab("w", "iii","uu");
Tab O = Tab("j", "mmm","dd");
Tab P = Tab("pp", "u","e");
Tab Q = Tab("x", "qqq","v");
Tab R = Tab("oo", "q","c");
Tab S = Tab("ggg", "dd","h");
Tab T = Tab("hhh", "vv","s");
Tab U = Tab("ee", "t","oo");
Tab V = Tab("zz", "q","qqq");
Tab W = Tab("ww", "lll","ooo");
Tab X = Tab("j", "gg","y");
Tab Y = Tab("u", "u","b");
Tab Z = Tab("y", "y","uu");
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.newTab(A);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.newTab(B);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(C);
d.changeActiveTabTo(1);
a.changeActiveTabTo(3);
b.newTab(D);
c.newTab(E);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
b.newTab(F);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.newTab(G);
d.changeActiveTabTo(4);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(H);
b.newTab(I);
c.newTab(J);
d.moveActiveTabTo(4);
a.moveActiveTabTo(0);
b.newTab(K);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.newTab(L);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(M);
d.moveActiveTabTo(3);
a.changeActiveTabTo(1);
b.newTab(N);
c.moveActiveTabTo(1);
d.newTab(O);
a.changeActiveTabTo(3);
b.newTab(P);
c.moveActiveTabTo(3);
d.newTab(Q);
a.newTab(R);
b.changeActiveTabTo(0);
c.newTab(S);
d.changeActiveTabTo(0);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(T);
d.newTab(U);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(4);
b.newTab(V);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
b.newTab(W);
c.moveActiveTabTo(3);
d.newTab(X);
a.newTab(Y);
b.newTab(Z);
c.changeActiveTabTo(5);
d.newTab(Z);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(0);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
d.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
d.changeActiveTabTo(2);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
d.moveActiveTabTo(2);
b.moveActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(2);
c.changeActiveTabTo(2);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
a.moveActiveTabTo(0);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
c.moveActiveTabTo(5);
b.changeActiveTabTo(4);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
c.changeActiveTabTo(3);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.moveActiveTabTo(0);
d.changeActiveTabTo(0);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->switchToWindow(1);
brows->closeEmptyWindows();
brows->getWindow(4);
brows->newWindow();
brows->print();
brows->newWindow();
brows->switchToWindow(4);
brows->print();
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->getWindow(0);
brows->switchToWindow(6);
brows->getWindow(1);
brows->closeEmptyWindows();
brows->getWindow(2);
brows->closeWindow();
brows->getWindow(9);
brows->getWindow(9);
brows->switchToWindow(1);
brows->getWindow(8);
brows->switchToWindow(4);
brows->newWindow();
brows->closeWindow();
brows->getWindow(5);
brows->getWindow(1);
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(6);
brows->switchToWindow(1);
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->newWindow();
brows->switchToWindow(0);
brows->closeWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->print();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->print();
brows->getWindow(0);
brows->switchToWindow(0);
brows->switchToWindow(9);
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->switchToWindow(5);
brows->switchToWindow(5);
brows->closeWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->closeWindow();
brows->closeWindow();
brows->switchToWindow(4);
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->getWindow(4);
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(3);
brows->switchToWindow(9);
brows->newWindow();
brows->switchToWindow(5);
brows->getWindow(0);
brows->switchToWindow(6);
brows->print();
brows->print();
brows->getWindow(8);
brows->newWindow();
brows->getWindow(3);
brows->switchToWindow(6);
brows->closeWindow();
brows->switchToWindow(1);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(4);
brows->newWindow();
brows->print();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(0);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->getWindow(3);
brows->closeWindow();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
delete brows;
}
	return 0;
}