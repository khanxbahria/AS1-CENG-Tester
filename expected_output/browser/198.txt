The active tab is tab 0




The active tab is tab 1
ddd
yy
fff

d
aaa
h

u
u
b

cc
ll
dd





q
oo
c

The active tab is tab 0




The active tab is tab 0
ss
jj
iii

j
sss
r

q
zz
qqq

lll
ww
ooo





y
y
uu

nn
ff
d

iii
w
uu

u
pp
e

uu
ll
eee

The active tab is tab 1
jj
iii
tt

g
ll
xxx

vv
hhh
s

sss
uu
q

ooo
uu
ee

dd
ggg
h

uu
sss
jj





The active tab is tab 0
zz
xxx
p





mmm
j
dd

qqq
x
v

t
ee
oo

gg
j
y

y
y
uu

The active tab is tab 0
ss
jj
iii

j
sss
r

q
zz
qqq

lll
ww
ooo





y
y
uu

nn
ff
d

iii
w
uu

u
pp
e

uu
ll
eee

The active tab is tab 0




The active tab is tab 0




The active tab is tab 1
ddd
yy
fff

d
aaa
h

u
u
b

cc
ll
dd





q
oo
c

The active tab is tab 0




The active tab is tab 1
jj
iii
tt

g
ll
xxx

vv
hhh
s

sss
uu
q

ooo
uu
ee

dd
ggg
h

uu
sss
jj





The active tab is tab 0
zz
xxx
p





mmm
j
dd

qqq
x
v

t
ee
oo

gg
j
y

y
y
uu

The active tab is tab 34




ss
jj
iii

j
sss
r

q
zz
qqq

lll
ww
ooo





y
y
uu

nn
ff
d

iii
w
uu

u
pp
e

uu
ll
eee









ddd
yy
fff

d
aaa
h

u
u
b

cc
ll
dd





q
oo
c





jj
iii
tt

g
ll
xxx

vv
hhh
s

sss
uu
q

ooo
uu
ee

dd
ggg
h

uu
sss
jj





zz
xxx
p





mmm
j
dd

qqq
x
v

t
ee
oo

gg
j
y

y
y
uu

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 34




ss
jj
iii

j
sss
r

q
zz
qqq

lll
ww
ooo





y
y
uu

nn
ff
d

iii
w
uu

u
pp
e

uu
ll
eee









ddd
yy
fff

d
aaa
h

u
u
b

cc
ll
dd





q
oo
c





jj
iii
tt

g
ll
xxx

vv
hhh
s

sss
uu
q

ooo
uu
ee

dd
ggg
h

uu
sss
jj





zz
xxx
p





mmm
j
dd

qqq
x
v

t
ee
oo

gg
j
y

y
y
uu

The active tab is tab 0




The active tab is tab 2












Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 2












The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The browser is empty
The browser is empty
The active tab is tab 0




